embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/destination/LogWriterImpl.kt

```diff
@@ -28,6 +30,8 @@ internal class LogWriterImpl(
             .setSeverity(logEventData.severity.toOtelSeverity())
             .setSeverityText(logEventData.severity.name)
 
+        builder.setAttribute(logRecordUid, Uuid.getEmbUuid())
```

Comment:
what about crashes? that get the unityCrashId? will be overwritten, right?

Reply:
Yup - anythign from teh schema will overwrite anything that is set here. The native crash actually overrides the sessionId because we want the old one, not the current one